.github/workflows/CD.yml

```diff
@@ -32,3 +32,31 @@ jobs:
             pipenv install --deploy &&
             sudo systemctl restart syncm8.service
           '
+
+  frontend-deploy:
+    runs-on: ubuntu-latest
+    environment: prod
+    needs: backend-deploy
```

Comment:
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-requiring-dependent-jobs-to-be-successful